<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>156</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Display Unit</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="displayUnitCombo">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Hz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Counts</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="MagnitudeSpinBox" name="integrationTimeBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Integration</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Points to keep</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="pointsToKeepBox">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QCheckBox" name="ch0Check">
       <property name="text">
        <string>Ch 0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QCheckBox" name="ch1Check">
       <property name="text">
        <string>Ch 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="ch2Check">
       <property name="text">
        <string>Ch 2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="ch3Check">
       <property name="text">
        <string>Ch 3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="ch4Check">
       <property name="text">
        <string>Ch 4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="ch5Check">
       <property name="text">
        <string>Ch 5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="ch6Check">
       <property name="text">
        <string>Ch 6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="ch7Check">
       <property name="text">
        <string>Ch 7</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="adc0Check">
       <property name="text">
        <string>ADC 0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="adc1Check">
       <property name="text">
        <string>ADC 1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="adc2Check">
       <property name="text">
        <string>ADC 2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="adc3Check">
       <property name="text">
        <string>ADC 3</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MagnitudeSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>uiModules.MagnitudeSpinBox</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pointsToKeepBox</tabstop>
  <tabstop>integrationTimeBox</tabstop>
  <tabstop>displayUnitCombo</tabstop>
  <tabstop>ch0Check</tabstop>
  <tabstop>ch1Check</tabstop>
  <tabstop>ch2Check</tabstop>
  <tabstop>ch3Check</tabstop>
  <tabstop>ch4Check</tabstop>
  <tabstop>ch5Check</tabstop>
  <tabstop>ch6Check</tabstop>
  <tabstop>ch7Check</tabstop>
  <tabstop>adc0Check</tabstop>
  <tabstop>adc1Check</tabstop>
  <tabstop>adc2Check</tabstop>
  <tabstop>adc3Check</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
